autopush/jwt.py

```diff
@@ -78,6 +78,20 @@ def extract_signature(auth):
         return payload, encoded
 
     @staticmethod
+    def extract(token):
```

Comment:
Yeah, I was keeping with the original library which I am emulating, but no reason not to be clearer about things.